src/main/java/org/libsodium/jni/Sodium.java

```diff
@@ -737,10 +737,6 @@ public static int crypto_onetimeauth_poly1305_verify(byte[] h, byte[] in, int in
     return SodiumJNI.crypto_onetimeauth_poly1305_verify(h, in, inlen, k);
   }
 
-  public static int crypto_onetimeauth_poly1305_statebytes() {
```

Comment:
I'm not sure. It was auto generated from sodium.i. 

https://github.com/jedisct1/libsodium/commit/f1fdb2a763b89c6321a24307603fa10d552794c7

Is it possible I am using an older version of libsodium somehow? I can just remove this change if you'd like. It has nothing to do with my intended changes.